modules/build/src/test/scala/scala/build/tests/TestInputs.scala

```diff
@@ -44,7 +45,9 @@ final case class TestInputs(
         forcedWorkspace = forcedWorkspaceOpt.map(_.resolveFrom(tmpDir))
       )
       res match {
-        case Left(err)     => sys.error(err)
+        case Left(err) =>
+          Util.printException(err)
+          sys.error(err.getMessage)
```

Comment:
NIT Why not just throw the exception? Optionally wrapping it in `new Exception(…)`, so that the stack trace here is included in it:
```suggestion
          throw new Exception(err)
```